content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -222,17 +222,26 @@ You will still have to enable each option using the `TopologyManagerPolicyOption
 
 The following policy options exists:
 * `prefer-closest-numa-nodes` (beta, visible by default; `TopologyManagerPolicyOptions` and `TopologyManagerPolicyBetaOptions` feature gates have to be enabled).
-The `prefer-closest-numa-nodes` policy option is beta in Kubernetes {{< skew currentVersion >}}.
-
-If the `prefer-closest-numa-nodes` policy option is specified, the `best-effort` and `restricted`
-policies will favor sets of NUMA nodes with shorter distance between them when making admission decisions.
-You can enable this option by adding `prefer-closest-numa-nodes=true` to the Topology Manager policy options.
-By default, without this option, Topology Manager aligns resources on either a single NUMA node or
-the minimum number of NUMA nodes (in cases where more than one NUMA node is required). However,
-the `TopologyManager` is not aware of NUMA distances and does not take them into account when making admission decisions.
-This limitation surfaces in multi-socket, as well as single-socket multi NUMA systems,
-and can cause significant performance degradation in latency-critical execution and high-throughput applications if the
-Topology Manager decides to align resources on non-adjacent NUMA nodes.
+  The `prefer-closest-numa-nodes` policy option is beta in Kubernetes {{< skew currentVersion >}}.
+
+  If the `prefer-closest-numa-nodes` policy option is specified, the `best-effort` and `restricted`
+  policies will favor sets of NUMA nodes with shorter distance between them when making admission decisions.
+  You can enable this option by adding `prefer-closest-numa-nodes=true` to the Topology Manager policy options.
+  By default, without this option, Topology Manager aligns resources on either a single NUMA node or
+  the minimum number of NUMA nodes (in cases where more than one NUMA node is required). However,
+  the `TopologyManager` is not aware of NUMA distances and does not take them into account when making admission decisions.
+  This limitation surfaces in multi-socket, as well as single-socket multi NUMA systems,
+  and can cause significant performance degradation in latency-critical execution and high-throughput applications if the
+  Topology Manager decides to align resources on non-adjacent NUMA nodes.
+
+* `max-allowable-numa-nodes` (beta; `TopologyManagerPolicyOptions` and `TopologyManagerPolicyBetaOptions` feature gates have to be enabled)
```

Comment:
ok, this is not related to this very change because I see the same text at line 225: this is not completely correct. The BetaOptions gate is always enabled by design. Please see https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L822 and https://github.com/kubernetes/kubernetes/blob/master/pkg/features/kube_features.go#L1200 .
IOW, the gates are currently enabled by default so no action is needed by default